spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require "email_spec"
+require "email_spec/rspec"
```

Comment:
This gem relies on ActionMailer, which is part of Rails, so should go in the rails_helper.